### DISCUSSION TITLE
Where are they?

### DISCUSSION THREAD

**Discussion by darrenliuwei** [community/community#58615] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![nothing](https://github.com/community/community/assets/10508248/4f898ba7-1d69-44f8-a0bd-a3bde77425db)

**Comment by airtower-luna** [comment, marked as answer]
Looks like a ghost notification:

* #6874

You can use the API to mark the ghost notifications as read, I wrote [this script](https://gist.github.com/airtower-luna/4f371c7d8799632ba5b9ed855039a484) (it reads the token for authorization from the `GH_TOKEN` environment variable) when I had the same problem a while ago. That way you can at least get rid of the "unread notifications" marker. After about a day the notifications actually showed up in the inbox. :sweat_smile:
